task-3/services/user.ts

```diff
@@ -0,0 +1,63 @@
+import { Op } from "sequelize";
+
+import { User } from "../models/User";
+
+class UserService {
+  static async findAllUsers(id: string) {
+    const user = await User.findOne({
+      where: {
+        id: id,
+      },
+    });
+    return user;
+  }
+
+  static async updateUser(
+    id: string,
+    login: string,
+    password: string,
+    age: number
+  ) {
+    await User.update(
+      { login: login, password: password, age: age },
```

Comment:
Consider using separate variables for config objects - as usual, improves readability.